### ISSUE TITLE
Audio quality 0.1.7

### ISSUE THREAD

**Issue by Sebbo401** [Mazamars312/openfpga-pcengine-cd#32]
For some reason it seems like the audio quality is worse then in the versions before.

Does not sound like the crisp CD in the previous version.

**Comment by Sebbo401** [comment]
Checked if single bin files, converted with CDmage could cause the problem - but same sound problems with multi bin files and every game I tested so far:

Rondo of Blood
Ys
Nexzr
Ginga Fukei Densetsu Sapphire

**Comment by JayroJones** [comment]
Yeah, this definitely took a regression hit with 0.17, as I primarily use this as a CD player for music. Can't believe such a regression would be released like this without anyone developing it to catch it first.

**Comment by mattpannella** [comment]
maybe you should write it yourself then, dude.

**Comment by Sebbo401** [comment]
I set the audio timer as low as possible, makes it way better, but still some cracking in the audio.

**Comment by mattd121** [comment]
Can confirm, sound is messed up in this version. Might be why my updater skipped over this update despite it being out for a week.

**Comment by JoeTheRetro** [comment]
I just updated, same problem here. Things worked great on 0.1.6. Now audio is weird, and my game controls freak out and won't respond when I load a CD Engine game. The only way to fix this is to go into the menu and choose to load a new ISO, once doing that, the controls stop spazzing and they work correctly. I had read about audio issues, but I decided to update anyway against my better judgment.

**Comment by Sebbo401** [comment]
Hi, I could stop the controller problems by disabling the 6 button controller for 3 button games in the core settings.

Also setting the audio timer to as low as possible made the sound way better - not perfect, bit way better.

**Comment by Sebbo401** [comment]
> I just updated, same problem here. Things worked great on 0.1.6. Now audio is weird, and my game controls freak out and won't respond when I load a CD Engine game. The only way to fix this is to go into the menu and choose to load a new ISO, once doing that, the controls stop spazzing and they work correctly. I had read about audio issues, but I decided to update anyway against my better judgment. 

Hi, I could stop the controller problems by disabling the 6 button controller for 3 button games in the core settings.

Also setting the audio timer to as low as possible made the sound way better - not perfect, but way better.

**Comment by JoeTheRetro** [comment]
> > I just updated, same problem here. Things worked great on 0.1.6. Now audio is weird, and my game controls freak out and won't respond when I load a CD Engine game. The only way to fix this is to go into the menu and choose to load a new ISO, once doing that, the controls stop spazzing and they work correctly. I had read about audio issues, but I decided to update anyway against my better judgment.
> 
> Hi, I could stop the controller problems by disabling the 6 button controller for 3 button games in the core settings.
> 
> Also setting the audio timer to as low as possible made the sound way better - not perfect, but way better.

Thanks, yeah, I noticed the 6 button fix thing right when you posted. A shame about the audio. I'll just hang on 0.1.6 until the next update. I don't seem to have any problems on that one.

**Comment by JayroJones** [comment]
It's been months, half the year has gone by without a fix to the audio regressions. Seems they don't want to fix the core.

**Comment by Sebbo401** [comment]
> It's been months, half the year has gone by without a fix to the audio regressions. Seems they don't want to fix the core.

Hi! You can just use an older version of the core or set the audio timer as low as possible. This made the sound way better for me.

**Comment by mattpannella** [comment]
> It's been months, half the year has gone by without a fix to the audio regressions. Seems they don't want to fix the core.

have you considered fixing it yourself?

**Comment by reacondicionad0** [comment]
> I set the audio timer as low as possible, makes it way better, but still some cracking in the audio.

This seems to improve it for me too, Thanks a lot!

Not perfect but much better :)
